src/Twilio.TaskRouter/Model/Filter.cs

```diff
@@ -17,6 +17,12 @@ public Filter()
         [DataMember(EmitDefaultValue = false, Name="friendly_name")]
         public string FriendlyName { get; set; }
 
+        [DataMember(EmitDefaultValue = false, Name = "filter_friendly_name")]
+        public string FilterFriendlyName {
+            get { return null; }

```

Comment:
should we be returning `this.FriendlyName` here? it feels confusing that a user could set something that they cannot then get.